### PR TITLE
Added exporter for IP Access Lists

### PR DESCRIPTION
**Pull request by alexott**

### CODE REVIEW

File: exporter/importables.go [hunk 1]
```diff
@@ -845,4 +847,70 @@ var resourcesMap map[string]importable = map[string]importable{
...
 		},
 	},
+	"databricks_workspace_conf": {
+		Service: "workspace",
+		Name: func(d *schema.ResourceData) string {
+			return globalWorkspaceConfName
+		},
+		Import: func(ic *importContext, r *resource) error {
+			wsConfAPI := workspace.NewWorkspaceConfAPI(ic.Context, ic.Client)
+			keys := map[string]interface{}{"enableIpAccessLists": false}
```
Comment (nfx) [inline exporter/importables.go:857]: this is very limited wsconf exporter. add a way to add more fields there.

**Review by nfx — CHANGES_REQUESTED** [review]
please remove custom body for ip acls - we don't need `depends_on` attribute.

File: exporter/importables.go (hunk 1 above)
Comment (alexott) [reply exporter/importables.go:857]: I didn't want to go with more than officially documented

File: exporter/importables.go [hunk 2]
```diff
@@ -845,4 +847,70 @@ var resourcesMap map[string]importable = map[string]importable{
...
+					Data: ic.Resources["databricks_workspace_conf"].Data(
+						&terraform.InstanceState{
+							ID:         globalWorkspaceConfName,
+							Attributes: map[string]string{},
+						}),
+				})
+			}
+			return nil
+		},
+		Body: func(ic *importContext, body *hclwrite.Body, r *resource) error {
```
[the remark this answers is not in the document]
Comment (alexott) [reply exporter/importables.go:897]: ah, then it makes even simpler

File: exporter/importables.go (hunk 1 above)
Comment (nfx) [reply exporter/importables.go:857]: https://registry.terraform.io/providers/databrickslabs/databricks/latest/docs/resources/workspace_conf there are few more ;)

**Review by nfx — APPROVED** [review]

File: exporter/importables.go (hunk 1 above)
Comment (alexott) [reply exporter/importables.go]: added

**Pull request merged**